src/components/forms/NetworkDevicesForm/edit/EditNetworkDevicePanel.tsx

```diff
@@ -0,0 +1,288 @@
+              value={formik.values.network}
+              setValue={(value) => {
+                void formik.setFieldValue("network", value);
+
+                const selectedNetwork = managedNetworks.find(
+                  (n) => n.name === value,
+                );
+                const newNetworkAcls = getNetworkAcls(selectedNetwork);
+
+                if (!supportsNicDeviceAcls(selectedNetwork)) {
+                  void formik.setFieldValue("acls", "");
+                } else {
+                  void formik.setFieldValue("acls", newNetworkAcls.join(","));
+                }
+
+                if (!supportsStaticIPs(selectedNetwork)) {
+                  void formik.setFieldValue("ipv4", "");
+                  void formik.setFieldValue("ipv6", "");
+                }
+              }}
+              managedNetworks={managedNetworks}
+              id="network"
+              label="Network"
+              required
+              error={formik.touched.network ? formik.errors.network : undefined}
+            />
+
+            <label
+              className={classnames({
+                "u-text--muted": !supportsNicDeviceAcls(selectedNetwork),
+              })}
+              htmlFor={networkAclSelectorId}
+            >
+              ACLs
+            </label>
+            <NetworkAclSelector
+              project={project}
+              selectedAcls={selectedAcls}
+              setSelectedAcls={(selectedItems) => {
+                void formik.setFieldValue("acls", selectedItems.join(","));
+              }}
+              id={networkAclSelectorId}
+              inheritedAcls={networkAcls}
+              canSelectManualAcls={supportsNicDeviceAcls(selectedNetwork)}
+              help={getAclHelperText()}
+            />
+
+            {parentFormik.values.entityType === "instance" &&
+              selectedNetwork && (
+                <>
+                  <NetworkDeviceIPAddressEdit
+                    formik={formik}
+                    network={selectedNetwork}
+                    family="IPv4"
+                  />
+                  <NetworkDeviceIPAddressEdit
+                    formik={formik}
+                    network={selectedNetwork}
+                    family="IPv6"
+                  />
+                </>
+              )}
+          </Form>
+        </ScrollableContainer>
+      </SidePanel.Content>
+      <SidePanel.Footer className="u-align--right">
+        <Button appearance="base" onClick={handleCancel} type="button">
+          Cancel
+        </Button>
+        <ActionButton
+          appearance="positive"
+          loading={formik.isSubmitting}
+          disabled={
+            isCreatingOverride
+              ? !formik.isValid
+              : !formik.isValid || !formik.dirty
+          }
+          onClick={() => {
+            formik.handleSubmit();
+          }}
+          type="button"
+        >
+          {isCreatingOverride ? "Create override" : "Apply changes"}
```

Comment:
```suggestion
          "Apply changes"
```
I think in the mental model of the user this is the same action, and we should maybe present it equally as an edit nic device with "Apply changes". Wdyt?